### PR TITLE
Updated broken links

### PR DESCRIPTION
**Pull request by singhalashish16**
Fixed broken links mentioned in Issue
#2212 

- Updated  [perspective/packages/perspective/README.md](https://github.com/finos/perspective/blob/e89ae0ab2aca1d903a0d3738ee2e8d9f67c1c684/packages/perspective/README.md?plain=1#L7)

- Updated [docs/src/components/HomepageFeatures/python.md](https://github.com/finos/perspective/blob/master/docs/src/components/HomepageFeatures/python.md)

### CODE REVIEW

File: docs/src/components/HomepageFeatures/python.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Python
 
 \`perspective-python\`, built on the same C++ data engine used by the
-[WebAssembly version](https://perspective.finos.org/docs/md/js.html), implements
+[WebAssembly version](https://perspective.finos.org/docs/obj/perspective-python/), implements
```
Comment (sinistersnare) [inline docs/src/components/HomepageFeatures/python.md:4]: This seems like the wrong link. It used to link to a `/js` page but now `/python`

Comment (singhalashish16) [reply docs/src/components/HomepageFeatures/python.md:4]: The page demands link to be directed to perspective-python api doc.
The current https://perspective.finos.org/docs/md/js.html gives error.
If it shoud direct to another relevent link please give it.

Comment (sinistersnare) [reply docs/src/components/HomepageFeatures/python.md:4]: The [actual Python documentation](https://perspective.finos.org/docs/python/) has similar copy at the top, and links to [The JS docs](https://perspective.finos.org/docs/js/) (https://perspective.finos.org/docs/js/). So I think we should link to the JS docs there. 

However, it probably is a good idea to link to the Python docs here, probably make the `perspective-python` at the start of the paragraph a link to https://perspective.finos.org/docs/python/, and not the API docs themselves?

Regardless of that though, this should be a link to the JS docs, as the Python docs do.

**Comment by texodus** [comment]
Wrong links, as per @sinistersnare.  This is easier to just fix than invite further iteration, so closing.

**Pull request closed without merging**